src/main/resources/edu/cornell/kfs/pdp/businessobject/datadictionary/PayeeACHAccountExtractDetail.xml

```diff
@@ -0,0 +1,294 @@
+                <ref bean="PayeeACHAccountExtractDetail-createDate"/>
+                <ref bean="PayeeACHAccountExtractDetail-status"/>
+                <ref bean="PayeeACHAccountExtractDetail-retryCount"/>
+                <ref bean="PayeeACHAccountExtractDetail-employeeID"/>
+                <ref bean="PayeeACHAccountExtractDetail-netID"/>
+                <ref bean="PayeeACHAccountExtractDetail-lastName"/>
+                <ref bean="PayeeACHAccountExtractDetail-firstName"/>
+                <ref bean="PayeeACHAccountExtractDetail-paymentType"/>
+                <ref bean="PayeeACHAccountExtractDetail-balanceAccount"/>
+                <ref bean="PayeeACHAccountExtractDetail-completedDate"/>
+                <ref bean="PayeeACHAccountExtractDetail-bankName"/>
+                <ref bean="PayeeACHAccountExtractDetail-bankRoutingNumber"/>
+                <ref bean="PayeeACHAccountExtractDetail-bankAccountNumber"/>
+                <ref bean="PayeeACHAccountExtractDetail-bankAccountType"/>
+            </list>
+        </property>
+    </bean>
+
+    <!-- Attribute Definitions -->
+
+    <bean id="PayeeACHAccountExtractDetail-id"
+          parent="PayeeACHAccountExtractDetail-id-parentBean"/>
+    <bean id="PayeeACHAccountExtractDetail-id-parentBean" abstract="true" parent="AttributeDefinition"
+          p:name="id" p:forceUppercase="true" p:label="ACH Account Extract Detail Generated Identifier"
+          p:shortLabel="ACH Account Extract Detail ID" p:maxLength="10" p:validationPattern-ref="NumericValidation"
+          p:required="true" p:formatterClass="org.kuali.kfs.sys.businessobject.NoCommaCurrencyFormatter"
+          p:control-ref="HiddenControl"/>
+          
+    <bean id="PayeeACHAccountExtractDetail-createDate" parent="PayeeACHAccountExtractDetail-createDate-parentBean"/>
+    <bean id="PayeeACHAccountExtractDetail-createDate-parentBean" abstract="true" parent="GenericAttributes-genericTimestamp"
+          p:name="createDate" p:label="Create Date" p:shortLabel="Crt Dt" p:required="false"/>
+          
+    <bean id="PayeeACHAccountExtractDetail-status"
+          parent="PayeeACHAccountExtractDetail-status-parentBean"/>
+    <bean id="PayeeACHAccountExtractDetail-status-parentBean" abstract="true"
+          parent="AttributeDefinition" p:name="status"
+          p:label="Status" p:shortLabel="Status">
+        <property name="control">
+            <bean parent="SelectControlDefinition" p:valuesFinder-ref="achAccountExtractStatusValuesFinder"
+                  p:includeKeyInLabel="false"/>
+        </property>
+    </bean> 
+    
+    <bean id="PayeeACHAccountExtractDetail-retryCount" parent="PayeeACHAccountExtractDetail-retryCount-parentBean"/>
+    <bean id="PayeeACHAccountExtractDetail-retryCount-parentBean" abstract="true" parent="AttributeDefinition"
+          p:name="retryCount" p:label="Retry Count" p:shortLabel="Retry Cnt" p:maxLength="10"
+          p:validationPattern-ref="NumericValidation" p:required="true">
```

Comment:
I'm not certain whether the maintenance document will have problems if the majority of these fields are listed as required. However, I'm okay with them being left as required ones, and then we can change them accordingly when we proceed to the next phase of implementing this feature.

Reply:
Will remove the required property. Do we know of fields that are specifically required?

Reply:
The validation in PayeeACHAccountExtractServiceImpl should indicate what fields are required. I'm not sure whether that required-value setup should be enforced for these new maintenance documents, though.

Reply:
I'm fine with the current state of the "required" indicator in the latest revisions. If further changes are needed, we can handle that in Phase 3 of the feature's implementation.